tests/data/content_text_html_common.xml

```diff
@@ -0,0 +1,5 @@
+<feed xmlns="http://www.w3.org/2005/Atom">
+	<entry>
+		<content type="html" base="http://example.com/blog/" lang="en_US">&lt;p&gt;Entry content&lt;/p&gt;&lt;a href="2021-05-51/article.html"&gt;Read more&lt;/a&gt;</content>
```

Comment:
If I understand XML namespacing correctly, `base` here comes from the Atom namespace, which is incorrect. I think `xml:base` in the Atom spec means that the attribute should come from the XML namespace instead.

To fix this, I think two things have to be done:
1. change `base` to `xml:base`
2. add `<?xml version="1.0">` at the start of the document

I'm not 100% sure on this though, I have a very vague idea of how XML namespaces are pulled together. It might well be that the Atom namespace inherits the XML namespace, or re-publishes it under the `xml` prefix.

Reply:
You are right. `base` and `lang` should be prefixed with `xml:`.

Thanks for a review.

Reply:
BTW, according to the [spec](https://www.w3.org/TR/xml-names/#xmlReserved), it is always `xml:`.